src/screens/DataApi/DataCoverageTable.tsx

```diff
@@ -0,0 +1,172 @@
+  { displayName: 'ICU capacity used', field: 'metrics.icuCapacityRatio' },
+  { displayName: 'Hospitalization usage', field: 'metrics.icuCapacityRatio' },
+  { displayName: 'Deaths', field: 'actuals.deaths' },
+  { displayName: 'Total testing volume', field: 'actuals.positiveTests' },
+];
+
+export const FIELDS_PRETTY_NAMES = FIELDS.map(field => {
+  const record = COVERAGE_SUMMARY.find(record => field.field === record.name);
+  if (record) {
+    return {
+      ...record,
+      ...{ name: field.displayName },
+    };
+  }
+
+  return null;
+}).filter((record): record is MetricCoverage => record !== null);
+
+export interface CoverageDetails {
+  totalRegions: number;
+  regionsAvailable: number;
+  totalStates: number;
+  statesAvailable: number;
+}
+
+export interface MetricCoverage {
+  name: string;
+  state: CoverageDetails;
+  county: CoverageDetails;
+  metro: CoverageDetails;
+}
+
+const TextWithTooltip: React.FC<{ text: string; tooltipContent: string }> = ({
+  text,
+  tooltipContent,
+}) => {
+  return (
+    <TextWithTooltipContainer>
+      <UnderlinedText align="center">
+        {text}
+        <InfoTooltip trackOpenTooltip={() => {}} title={tooltipContent} />
+      </UnderlinedText>
+    </TextWithTooltipContainer>
+  );
+};
+
+const tooltipText = (coverage: CoverageDetails, regionType: RegionType) => {
+  switch (regionType) {
+    case RegionType.COUNTY:
+      return (
+        `Supported in ${coverage.regionsAvailable} counties (of ${coverage.totalRegions}), ` +
+        `or ${coverage.statesAvailable} states and territories (of ${coverage.totalStates}).`
+      );
+    case RegionType.MSA:
+      return `Supported in ${coverage.regionsAvailable} metros (of ${coverage.totalRegions}).`;
+    case RegionType.STATE:
+      return `Supported in ${coverage.regionsAvailable} states (of ${coverage.totalRegions}).`;
+  }
+};
+
+const Status: React.FC<{ status: CoverageDetails; level: RegionType }> = ({
+  status,
+  level,
+}) => {
+  if (status.regionsAvailable === 0) {
+    return (
+      <TextWithTooltip text="No data" tooltipContent="No data available." />
+    );
+  }
+
+  const availableRatio = status.regionsAvailable / status.totalRegions;
+
+  if (availableRatio >= 0.95) {
+    return <CheckIcon color="secondary" />;
+  }
+
+  const tooltipContent = tooltipText(status, level);
+  if (availableRatio >= 0.6 || level === RegionType.MSA) {
+    return (
+      <TextWithTooltip
+        text={`~${formatPercent(availableRatio, 0)}`}
+        tooltipContent={tooltipContent}
+      />
+    );
+  }
+
+  const statesText = pluralize(status.statesAvailable, 'State', 'States');
+  return (
+    <TextWithTooltip
+      text={`${status.statesAvailable} ${statesText}`}
+      tooltipContent={tooltipContent}
+    />
+  );
+};
+
+const CoverageRow: React.FC<{ row: MetricCoverage }> = ({ row }) => {
+  return (
+    <MetricTableRow key={row.name}>
+      <MetricTableCell>{row.name}</MetricTableCell>
+      <AvailabilityTableCell>
```

Comment:
Remember to add keys on children items for the same type

```tsx
<AvailabilityTableCell key="state-cell">
  {/* ... */}
</AvailabilityTableCell />
```

Reply:
hmm I didn't think you need for this case - I've only been able to find that this is true for lists/iterators.